### PR TITLE
Features/gettxoutproof

### LINKED ISSUE #168: Add "gettxoutproof" endpoint
This is useful for claiming pegins with elements

gettxoutproof ["txid",...] ( "blockhash" )

Returns a hex-encoded proof that "txid" was included in a block.

NOTE: By default this function only works sometimes. This is when there is an
unspent output in the utxo for this transaction. To make it always work,
you need to maintain a transaction index, using the -txindex command line option or
specify the block in which the transaction is included manually (by blockhash).

Arguments:
1. txids          (json array, required) A json array of txids to filter
     [
       "txid",    (string) A transaction hash
       ...
     ]
2. blockhash      (string, optional) If specified, looks for txid in the block with this hash

Result:
"data"           (string) A string that is a serialized, hex-encoded data for the proof.

### PR DESCRIPTION
**Pull request by filecommander**
for review

### CODE REVIEW

**Review by Kexkey — CHANGES_REQUESTED** [review]

File: proxy_docker/app/script/blockchainrpc.sh
Comment (filecommander) [reply proxy_docker/app/script/blockchainrpc.sh]: done

File: proxy_docker/app/tests/tests.sh
Comment (Kexkey) [inline proxy_docker/app/tests/tests.sh]: This is cool!  Could you please move it above in the "Testing deriveindex" block?  That's where it actually goes...

Here in the "Testing derivepubpath" block, we already provide the xpub in the call...

File: proxy_docker/app/tests/tests.sh
Comment (Kexkey) [inline proxy_docker/app/tests/tests.sh]: Nothing to do with your PR but it looks like CLN's newaddr used to return an `address` field but now the field name is `bech32` (default)... could you change it while you're in there?  Thanks!

**Review by Kexkey — CHANGES_REQUESTED** [review]

**Comment by Kexkey** [comment]
I'll merge right after

File: proxy_docker/app/tests/tests.sh
Comment (filecommander) [reply proxy_docker/app/tests/tests.sh]:
```
Testing ln_newaddr...

response={"bech32":"bcrt1q5k5nxkp8tkfgd3ah35p5e6nh4z6m9c2ualn3xf"}
address="bcrt1q5k5nxkp8tkfgd3ah35p5e6nh4z6m9c2ualn3xf"
Tested ln_newaddr.

```

File: proxy_docker/app/tests/tests.sh
Comment (filecommander) [reply proxy_docker/app/tests/tests.sh]: Testing deriveindex...

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 10.0.1.36:8888...
* Connected to proxy (10.0.1.36) port 8888 (#0)
> GET /deriveindex/25-30 HTTP/1.1
> Host: proxy:8888
> User-Agent: curl/7.80.0
> Accept: */*
> 
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Content-Type: application/json
< Content-Length: 16
< 
{ [16 bytes data]
100    16  100    16    0     0     10      0  0:00:01  0:00:01 --:--:--    10
* Connection #0 to host proxy left intact
response={"addresses":[]}
Lenght: [0]

In setup, make sure you set your default xpub key to upub5GtUcgGed1aGH4HKQ3vMYrsmLXwmHhS1AeX33ZvDgZiyvkGhNTvGd2TA5Lr4v239Fzjj4ZY48t6wTtXUy2yRgapf37QHgt6KWEZ6bgsCLpb
and default derivation to 0/n

**Review by Kexkey — APPROVED** [review]

**Pull request merged**
